### PR TITLE
LPS-152472 Update structure version on Form Instance to prevent null pointer

### PR DESCRIPTION
**Pull request by leticia-maciel**
Related Issue: https://issues.liferay.com/browse/LPS-152472

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @leticia-maciel, can you please check if these failures are related to your changes? Thanks

**Comment by leticia-maciel** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java [hunk 1]
```diff
@@ -337,6 +337,13 @@ public DDMFormInstance getFormInstance() {
 
 					_ddmFormInstance.setSettings(
 						latestApprovedDDMFormInstanceVersion.getSettings());
+
+					DDMStructureVersion structureVersion =
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:341]: @leticia-maciel I would rename this to `ddmStructureVersion` but I'll forwart as it is so we don't have to run CI again.

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**